src/Angor/Client/Pages/View.razor

```diff
@@ -1291,5 +1374,21 @@
             StateHasChanged();
         }
     }
+
+    public class SignatureItem
```

Comment:
delete this no need for this class here, delete all code that is not part of the method I mentioned